packages/dds/tree/src/test/shared-tree/undo.spec.ts

```diff
@@ -482,3 +607,42 @@ export function createCheckout(json: JsonCompatible[], attachTree: boolean): ITr
 }
 
 let temp: unknown;
+
+/**
+ * Schema Builder
+ */
+const builder = new SchemaFactory("shared-tree-test");
```

Comment:
Nit: I think we've been moving away from the "builder" naming to "factory". We used to have a `SchemaBuilder`, which is where the "builder" naming came from (I believe). But that no longer exists and isn't what's being used here.